core/new-gui/src/app/workspace/service/cache-workflow/workflow-cache.service.ts

```diff
@@ -25,33 +24,40 @@ import { localGetObject, localSetObject } from '../../../common/util/storage';
 @Injectable({
   providedIn: 'root'
 })
-export class CacheWorkflowService {
+export class WorkflowCacheService {
```

Comment:
We need some documents to explain the relationship between this and workflow persist sevice, and how each was used in different scenarios. 

The logic a bit unclear to me right now. For example, when a url with workflow ID is loaded, when a url without workflow id is loaded, etc..

Also I found workflow persist service is reading from workflow cache service, instead of directly reading from the workflow action service. Although both ways are fine, some documentation is good

Reply:
added doc to `workspace.component.ts`.